lib/ViewModels/ChartPanelViewModel.js

```diff
@@ -49,6 +57,16 @@ var ChartPanelViewModel = function(options) {
 
     updateMaxDimensions();
 
+    knockout.getObservable(this, 'isVisible').subscribe(function(isVisible) {
+        var cesiumContainer = this.terria.currentViewer.viewer.container;
```

Comment:
Ha! That's what you removed in the last commit! I've just poked around in terriajs's Leaflet class and found this:
`map.getContainer()`, ie. we can use `this.terria.currentViewer.map.getContainer()` here - this returns the DOM element we need.